acceptance/tests/vault/vault_test.go

```diff
@@ -164,6 +165,11 @@ func TestVault(t *testing.T) {
 		"global.gossipEncryption.secretName": "consul/data/secret/gossip",
 		"global.gossipEncryption.secretKey":  "gossip",
 
+		"ingressGateways.enabled":               "true",
+		"ingressGateways.defaults.replicas":     "1",
+		"terminatingGateways.enabled":           "1",
```

Comment:
```suggestion
		"terminatingGateways.enabled":           "true",
```

Reply:
copy/pasta spotting legend! thank you :)